spec/helpers.lua

```diff
@@ -21,11 +22,47 @@ local dao = DAOFactory(conf)
 -- make sure migrations are up-to-date
 --assert(dao:run_migrations())
 
+--------------------
+-- Custom properties
+--------------------
+local admin_port = string.match(conf.admin_listen, ":([%d]+)$")
+local proxy_port = string.match(conf.proxy_listen, ":([%d]+)$")
+local ssl_proxy_port = string.match(conf.proxy_listen_ssl, ":([%d]+)$")
```

Comment:
Re: those are now included in the config table itself. See https://github.com/Mashape/kong/blob/refactor/cli/kong/conf_loader.lua#L290